resources/assets/js/core/components/typeahead.vue

```diff
@@ -142,6 +142,21 @@ export default {
                 }
             }
         },
+        escape(item) {
+            let escItem = { ...item };
+            let escape = function (match) {
+                const escape = {
+                    '&': '&amp;',
+                    '<': '&lt;',
+                    '>': '&gt;',
+                    '"': '&quot;',
+                    "'": '&#39;'
+                };
+                return escape[match];
+            };
+            escItem.name = escItem.name.replace(/[&<>"']/g, escape);
+            return escItem;
+        }
```

Comment:
Please implement this as an `escapeHtml` function in `core/utils.js` so it can be reused in the future.